docs/en/Configuration-Settings.md

```diff
@@ -8,31 +8,75 @@ priority: 1
 * Table of Contents
 {:toc}
 
-There are two types of configuration parameters for Alluxio:
+This page explains the configuration system of Alluxio and also provides recommendation on how customize the configuration
+for Alluxio users in different contexts.
 
-1. [Configuration properties](#configuration-properties) are used to configure the runtime settings
-of Alluxio system, and
-2. [System environment properties](#system-environment-properties) control the Java VM options to
-run Alluxio as well as some basic very setting.
+# Configuration in Alluxio
 
-# Configuration properties
+Alluxio runtime respects two sources of configuration settings:
```

Comment:
this page should comment on the priority of configuration settings? defaults < properties files < environment variables